pkg/features/kube_features.go

```diff
@@ -171,12 +171,24 @@ const (
 	// Enable pods to set sysctls on a pod
 	Sysctls featuregate.Feature = "Sysctls"
 
+	// owner @smarterclayton
+	// alpha: v1.16
+	//
+	// Enable legacy behavior to vary cluster functionality on the node-role.kubernetes.io labels. On by default (legacy), will be turned off in 1.18.
```

Comment:
"cluster functionality" is ambiguous - propose we list the functionality it turns off: "Allow Service LoadBalancer to respect the node-role.kubernetes.io label when determining node inclusion."   Or maybe rephrased in human :-)

Reply:
We generally don't get very descriptive in these gates.  The behavior we are removing is "using node-role label to change things".  Expectation is that people look at the use of the gates or docs, but not here.